stf_flasher_dummyid.patch
     o Graceful exit when running tests with no flasherboard or a
     dead flasherboard

Index: flasher_clock.c
===================================================================
RCS file: /home/icecube/cvsroot/stf/private/stf-apps/flasher_clock.c,v
retrieving revision 1.2.2.2
diff -u -r1.2.2.2 flasher_clock.c
--- flasher_clock.c	15 Mar 2005 23:00:05 -0000	1.2.2.2
+++ flasher_clock.c	13 Apr 2005 14:48:31 -0000
@@ -32,7 +32,7 @@
                            unsigned int * reset_time_us
                            ) {
 
-    char dummy_id[9] = "deadbeef";
+    static char dummy_id[9] = "deadbeef";
     *flasher_id = dummy_id;
 
     /* Make sure PMT is off */
Index: flasher_width.c
===================================================================
RCS file: /home/icecube/cvsroot/stf/private/stf-apps/flasher_width.c,v
retrieving revision 1.4.2.2
diff -u -r1.4.2.2 flasher_width.c
--- flasher_width.c	15 Mar 2005 23:00:05 -0000	1.4.2.2
+++ flasher_width.c	13 Apr 2005 14:48:38 -0000
@@ -71,7 +71,7 @@
     *failing_led = *missing_width = 0;
     *config_time_us = *reset_time_us = *valid_time_us = 0;
 
-    char dummy_id[9] = "deadbeef";
+    static char dummy_id[9] = "deadbeef";
     *flasher_id = dummy_id;
 
     /* Pedestal buffers -- only use channel 3 in this test */
